sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/BlobsCheckpointStore.cs

```diff
@@ -15,13 +16,13 @@ namespace Azure.Messaging.EventHubs.Processor
     ///   A storage blob service that keeps track of checkpoints and ownership.
     /// </summary>
     ///
-    public sealed class BlobsCheckpointStore : PartitionManager
+    internal sealed class BlobsCheckpointStore : PartitionManager
```

Comment:
Just FYI, https://github.com/Azure/azure-sdk-for-net/pull/8935#discussion_r351445126

Not sure, if you're planning to do in this PR.

Reply:
I'll try to do it here. We may leave it like this for the preview if it proves to be troublesome, but it should be pretty straightforward.

Reply:
Done!